src/modules/avformat/producer_avformat.c

```diff
@@ -623,6 +638,10 @@ static enum AVPixelFormat pick_pix_fmt( enum AVPixelFormat pix_fmt )
 #if defined(FFUDIV)
 	case AV_PIX_FMT_BAYER_RGGB16LE:
 		return AV_PIX_FMT_RGB24;
+#endif
+#ifdef USE_VAAPI
+	case AV_PIX_FMT_VAAPI:
```

Comment:
Can this be expanded for hwaccel support beyond just VAAPI? If so, then we change the define macro name to `USE_HWACCEL`